server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/JMAPApiRoutes.scala

```diff
@@ -147,26 +147,26 @@ class JMAPApiRoutes (val authenticator: Authenticator,
         .collectSeq())
   }
 
-  private def process(capabilities: Set[CapabilityIdentifier], mailboxSession: MailboxSession, invocation: InvocationWithContext) : SMono[InvocationWithContext] = {
-    SMono.defer(() => {
+  private def process(capabilities: Set[CapabilityIdentifier], mailboxSession: MailboxSession, invocation: InvocationWithContext) : SFlux[InvocationWithContext] = {
+    SFlux.fromPublisher(Flux.defer(() => {
```

Comment:
I think there is a `SFlux.defer` that exists... but maybe less convenient to use than the java version of it?

Reply:
I did not succeed to make it work...